tests/runtimes/simple-keyvalue/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "Apache-2.0"
 
 [dependencies]
 oasis-runtime-sdk = { path = "../../../runtime-sdk" }
-oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core", rev = "a1c9fe982c32d550e8eac530bf8e79402728d64a" }
+oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core", rev = "a509afa" }
```

Comment:
Why is this even needed? You should get it from `oasis_runtime_sdk::core` instead.